ws-client/src/client.rs

```diff
@@ -488,6 +478,17 @@ async fn send_unsubscribe_request(
 	request: RequestMessage,
 ) {
 	if let Err(e) = sender.start_request(request, manager).await {
-		log::error!("send unsubscribe request failed: {:?}", e);
+		log::error!("Send unsubscribe request failed: {:?}", e);
 	}
 }
+
+fn build_unsubscribe_request(
```

Comment:
"request" is an inflated term in this library (any networking library I guess) and I wonder if a better name would be `build_unsubscribe_message`? Like: use "request" for actual users making a call for data from some server and "message" for internal communication between components?